server/api/room.js

```diff
@@ -13,13 +13,15 @@ router.get('/', async (req, res, next) => {
   }
 });
 
-// GET - /api/room/create/
```

Comment:
Decided to rename some of these routes to be the most RESTful possible. so:
- POSTing to /api/room creates a room.
- GETting /api/room/:roomId gets that room's info